src/ir/context.rs

```diff
@@ -1070,6 +1071,7 @@ impl<'ctx> BindgenContext<'ctx> {
                 let name = item.canonical_path(self)[1..].join("::");
                 debug!("whitelisted_items: testing {:?}", name);
                 match *item.kind() {
+                    ItemKind::ObjCInterface(_) => true, // TODO:
```

Comment:
Frankly, I wasn't sure, and I think I just looked at the try_parse! and thought ‘thats what I need’.

Forward declaration is a bit weird, using `@class Foo;`, this gives a cursor type CXCursor_ObjCClassRef, have not looked at that more closely.

One other interesting bit is protocols, those are for extending more methods for existing interface. Those come in as CXCursor_ObjCCategoryDecl, so I guess they could be handled completely separate (and generate their own trait)